internal/quickstart/show_sdk_instructions.go

```diff
@@ -16,10 +16,9 @@ import (
 	"ldcli/internal/sdks"
 )
 
-const (
-	viewportWidth  = 80
-	viewportHeight = 30
-)
+// stepCountHeight is the approximate height of the current step value shown from the container and
+// is used to calculate height of viewport.
+const stepCountHeight = 4
```

Comment:
we could also maybe do `lipgloss.Height("Step X of Y\n")` to account for terminal size weirdness but this probably works too